### PR TITLE
Blog updates

### PR DESCRIPTION
**Pull request by PaulMichaelArmstrong**
• Added March community meeting recap
• Added Jason and Alejandro as authors
• Added author image for Alex

Signed-off-by: Paul Armstrong <paul@buoyant.io>

### CODE REVIEW

File: linkerd.io/content/blog/march-linkerd-community-meeting.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+title = "March Linkerd Community Meeting Recap"
+description = "In this month's community meeting, we discussed what's on the roadmap, what's new with Linkerd 2.10, and announced the Linkerd Hero of the month."
+featured = false
++++
+
+Missed the last Linkerd Community Meeting? No worries. Here’s a  recap plus the recording.
+
+## News & updates
+
+Linkerd 2.10 is now available and is the lightest it has ever been. Many of the previous defaults are now extensions, providing users with a lot more control. We have also released the 21.3.4 edge release – please check it out and provide feedback! Another big news is that Linkerd submitted its CNCF graduation application. Stay tuned for more news on that!
```
Comment (cpretzer) [inline linkerd.io/content/blog/march-linkerd-community-meeting.md:16]: What do you think about moving the graduation bit to its own line? It's significant and feels a little buried here.

Also, it should read `Another really big piece of news...`

File: linkerd.io/content/blog/march-linkerd-community-meeting.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+## Linkerd roadmap
+
+We are setting the stage for policy in 2.11 and will continue consolidating control plane components. Join our next community meeting for more details.
```
Comment (cpretzer) [inline linkerd.io/content/blog/march-linkerd-community-meeting.md:20]: Do we have a link to sign up for the next community meeting? That would be great to have here.

File: linkerd.io/content/blog/march-linkerd-community-meeting.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+## What's new with Linkerd 2.10
+
+In 2.10, we've modularized the control plane. Users can now choose between multiple opt-in extensions; gaining more control over their service mesh implementation and making Linkerd even more lightweight. We also added opaque ports to extend Linkerd’s coverage to certain situations that are incompatible with protocol detection. Finally, we extended multi-cluster to all TCP connections. Learn more in our blog post on the release.
```
Comment (cpretzer) [inline linkerd.io/content/blog/march-linkerd-community-meeting.md:24]: Link to the blog post here

File: linkerd.io/content/blog/march-linkerd-community-meeting.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+## Linkerd Heroes
+
+This month we nominated four great Linkerd Hero candidates: Fredrik Klingenberg and Henry Hagnas for sharing their amazing Linkerd story; Christian Hüning for always jumping in on the Linkerd Slack to answer questions and help fellow Linkerd adopters; and Manyak for his great code contributions. As always, it was a close call but in the end, the community voted for Manyak – congrats!  👏👏👏
```
Comment (cpretzer) [inline linkerd.io/content/blog/march-linkerd-community-meeting.md:28]: This should be `Mayank`

**Review by cpretzer — CHANGES_REQUESTED** [review]
Couple of small notes here

File: linkerd.io/content/blog/march-linkerd-community-meeting.md (hunk 1 above)
Comment (PaulMichaelArmstrong) [reply linkerd.io/content/blog/march-linkerd-community-meeting.md:16]: @CathPag

File: linkerd.io/content/blog/march-linkerd-community-meeting.md (hunk 2 above)
Comment (PaulMichaelArmstrong) [reply linkerd.io/content/blog/march-linkerd-community-meeting.md:20]: @CathPag

File: linkerd.io/content/blog/march-linkerd-community-meeting.md (hunk 3 above)
Comment (PaulMichaelArmstrong) [reply linkerd.io/content/blog/march-linkerd-community-meeting.md:24]: @CathPag CTA at the end? "Read more about it here."

File: linkerd.io/content/blog/march-linkerd-community-meeting.md (hunk 4 above)
Comment (CathPag) [reply linkerd.io/content/blog/march-linkerd-community-meeting.md:28]: Yes! Thanks for the catch, @cpretzer

File: linkerd.io/content/blog/march-linkerd-community-meeting.md (hunk 3 above)
Comment (CathPag) [reply linkerd.io/content/blog/march-linkerd-community-meeting.md:24]: Oops, did I forget the link? It should be: Learn more in our [blog post on the release](https://linkerd.io/2021/03/11/announcing-linkerd-2.10/).

Comment (PaulMichaelArmstrong) [reply linkerd.io/content/blog/march-linkerd-community-meeting.md:24]: No. I am moving to fast and didn't realize that was already in there and I missed hyperlinking it.

File: linkerd.io/content/blog/march-linkerd-community-meeting.md (hunk 2 above)
Comment (PaulMichaelArmstrong) [reply linkerd.io/content/blog/march-linkerd-community-meeting.md:20]: Not yet. I am checking with Stephanie about it.

**Review by cpretzer — APPROVED** [review]
Once we get the link for the April meeting, this will be good to go

**Comment by PaulMichaelArmstrong** [comment]
@cpretzer They are wanting to try the platform before posting the next meeting. Can we merge and we'll add the link later?

**Comment by cpretzer** [comment]
> @cpretzer They are wanting to try the platform before posting the next meeting. Can we merge and we'll add the link later?

Mergified! 😄

**Pull request merged**